modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -579,6 +582,15 @@ private void _initSystemObjectDefinition(
 	@Reference
 	private GroupLocalService _groupLocalService;
 
+	@Reference
+	private JSONFactory _jsonFactory;
+
+	@Reference
+	private Language _language;
+
+	@Reference
+	private ObjectActionEngine _objectActionEngine;
```

Comment:
@diogomelo09, are we using `_objectActionEngine` and `_jsonFactory` in this class?